### ISSUE TITLE
[GlobalStep] Default store location is set to "United Kingdom" instead of "United States".

### ISSUE THREAD

**Issue by gglobalstep** [woocommerce/woocommerce#29730]
## Bug Description:
The Default store location is set to "United Kingdom" instead of "United States".

## Test Instructions:
https://github.com/woocommerce/woocommerce/wiki/Release-Testing-Instructions-WooCommerce-5.3#set-usca-as-the-default-store-location

## Environment:
Woocommerce Version : WooCommerce 5.2.2 and 5.3.0 Beta
WordPress version: (v5.7)

**PC:**
Windows 10, Mac 10.14.6
Chrome(Version 89.0.4389.90) 
Firefox(Version 87.0)
Safari: v13.1.1

## Steps To Reproduce:
1. Create any test site using JN site.
1. Install and activate all the required plugins 
1. Go to the onboarding setup wizard.
1. Click the link at the bottom to skip it.
1. Go to /wp-admin/admin.php?page=wc-settings.
1. Observe that default store location is set to "United Kingdom"

## Actual Result:
The Default store location is set to "United Kingdom" instead of "United States".

## Expected Result:
The Default store location should be set to "United States".

## ScreenShot:
![#29730](https://user-images.githubusercontent.com/41110392/115590634-d38bc200-a2ee-11eb-8f51-65ac30b4fa96.png)

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://secure-pintail.jurassic.ninja
Site address (URL): https://secure-pintail.jurassic.ninja
WC Version: 5.2.2
REST API Version: ✔ 5.2.2
WC Blocks Version: ✔ 4.7.2
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 2.1.5
Log Directory Writable: ✔
WP Version: 5.7.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.46 (Unix) OpenSSL/1.0.2g
PHP Version: 7.4.16
PHP Post Max Size: 1 GB
PHP Time Limit: 30
PHP Max Input Vars: 5000
cURL Version: 7.47.0
OpenSSL/1.0.2g

SUHOSIN Installed: –
MySQL Version: 5.7.33-0ubuntu0.16.04.1-log
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.2.2
WC Database Prefix: wp_
Total Database Size: 3.70MB
Database Data Size: 2.32MB
Database Index Size: 1.38MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 1.48MB + Index: 0.03MB + Engine InnoDB
wp_postmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_posts: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 1
page: 6
post: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (4) ###

Query Monitor: by John Blackbourn – 3.6.7
Companion Plugin: by Osk – 1.18
Jetpack by WordPress.com: by Automattic – 9.6.1
WooCommerce: by Automattic – 5.2.2

### Inactive Plugins (2) ###

Akismet Anti-Spam: by Automattic – 4.1.9
Hello Dolly: by Matt Mullenweg – 1.7.2

### Dropin Plugins (1) ###

db.php: Query Monitor Database Class

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/
Checkout: #8 - /checkout/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty-One
Version: 1.3
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 1
Oldest: 2021-04-21 12:08:56 +0000
Newest: 2021-04-21 12:08:56 +0000


### Status report information ###

Generated at: 2021-04-21 16:34:12 +00:00
`
```
</details>

**Comment by juliaamosova** [comment]
Hi @gglobalstep,

Thank you for raising the issue! 

There are a few things I noticed on my end which are slightly different from your experience. 

## Case 1:

Upgrading from the previous version of WooCommerce to 5.3 beta and testing. 

In this case, I took the following steps: 

- Installed WooCommerce 5.2.2 
- Skipped onboarding wizard after WooCommerce activation 
- Installed WooCommerce Beta Tester plugin
- Updated WooCommerce to 5.3 beta using WooCommerce Beta Tester plugin

Have you followed the same steps in your testing?

In this case, the default country remains `United Kingdom` which I think is expected because in WooCommerce 5.2.2 the default country was `United Kingdom`. The default country `United States` was introduced in WooCommerce 5.3 only which makes me think that we can only expect the default country to be `United States` for new installations starting WooCommerce 5.3.

## Case 2:

Testing using WooCommerce 5.3 beta zip. 

In this case, I took the following steps:

- Downloaded WooCommerce 5.3 beta zip from [here](https://github.com/woocommerce/woocommerce/releases/tag/5.3.0-beta.1)
- Installed and activated WooCommerce on the site
- Onboarding wizard started automatically
- Skipped onboarding wizard right at the beginning
- Navigated to `/wp-admin/admin.php?page=wc-settings` 
- Default country was `United States` - **as expected**
- Currency was set to `Pound sterling` - **not as expected**. I expected the currency to be set to `US Dollar` by default.

Please see the gif below documenting my experience:

http://g.recordit.co/oFI78wPVLf.gif

![us_dollar](https://user-images.githubusercontent.com/19143190/115767547-46408000-a377-11eb-8b96-de8e5b4b3edb.png)

---

The bug that is to be fixed here is the currency part. If `United States` is set as the default country, the currency should be set to `US Dollar` by default as well. 

Assigning this issue a 5.3 milestone as this feature is being introduced in the WC 5.3 release and currently is not working as expected.

**Comment by claudiosanches** [comment]
@juliaamosova We didn't changed the default current yet, so not sure if it's a bug, but it should for sure select `US - California` by default: https://github.com/woocommerce/woocommerce/pull/29654

**Comment by juliaamosova** [comment]
>@juliaamosova We didn't changed the default current yet, so not sure if it's a bug, but it should for sure select US - California by default: #29654

@claudiosanches thanks! This bug was filed as a result of WooCommerce 5.3 beta (where https://github.com/woocommerce/woocommerce/pull/29654 is a part of) testing. Because of this, I think we need to fix it for 5.3 release. 

https://github.com/woocommerce/woocommerce/wiki/Release-Testing-Instructions-WooCommerce-5.3#set-usca-as-the-default-store-location
